_layouts/paginated.html

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+---
+<div class="home">
+    {%- if page.title -%}
+    <h1 class="page-heading">{{ page.title }}</h1>
+    {%- endif -%}
+
+    <!-- Pagination links -->
+    <div class="pagination">
+        {% if paginator.previous_page %}
+        <a href="{{ paginator.previous_page_path }}" class="previous">Previous</a>
+        {% else %}
+        <span class="previous">Previous</span>
+        {% endif %}
+        <span class="page_number ">Page: {{ paginator.page }} of {{ paginator.total_pages }}</span>
+        {% if paginator.next_page %}
+        <a href="{{ paginator.next_page_path }}" class="next">Next</a>
+        {% else %}
+        <span class="next ">Next</span>
+        {% endif %}
+    </div>
+
+    <!-- This loops through the paginated posts -->
+    {% for post in paginator.posts %}
+    <h1><a href="{{ post.url }}">{{ post.title }}</a></h1>
+    <p class="author">
+        <span class="date">{{ post.date }}</span>
+    </p>
+    <div class="content">
+        {{ post.content }}
+    </div>
+    {% endfor %}
```

Comment:
May use `home` to display posts instead : [minima/home](https://github.com/jekyll/minima/blob/master/_layouts/home.html)